src/components/buttons/AddNodeButton.vue

```diff
@@ -33,11 +34,12 @@ export default class AddNodeButton extends Vue {
     if (NodeType === undefined) {
       console.error(`Undefined Node Type: ${this.node}`);
     } else {
-      const node = editor.addNode(new NodeType());
+      const node = editor.addNode(new NodeType(this.options));
       const { scaling, panning } = EditorManager.getInstance().viewPlugin;
       const { x: xPanning, y: yPanning } = panning;
       node.position.x = (window.innerWidth / (3 * scaling)) - xPanning;
       node.position.y = (window.innerHeight / (3 * scaling)) - yPanning;
+      this.$emit('node-created', node);
```

Comment:
I don't think we are using this. Maybe I am missing something?

Reply:
Yeah but we might want to in the future, it's nice to have imo. I used it in some of the implementations, although you're right, it's not used atm.

Reply:
We should remove it if it not being used. We can re-add later if needed very easily